src/trades/TradesLayouts.js

```diff
@@ -26,22 +28,43 @@ export default class TradesLayouts extends Component {
 
     render() {
         const { actions, assetsIsOpen, currency, layoutN, trades, ticksForAllSymbols, contracts } = this.props;
+        const tradesCount = trades.length;
+        const contSize = tradesCount > 0 ? layoutData[tradesCount][layoutN] : undefined;
```

Comment:
I am not sure what this code is, can you explain?
Also, can you think of another way to implement the styling without touching this file?